Frontend/UpFluxAutomation/Flows/FlowFactory.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Playwright;
+using System;
+using UpFluxAutomation.Abstractions;
+using UpFluxAutomation.Helpers;
+using UpFluxAutomation.Steps;
+
+namespace UpFluxAutomation.Flows
+{
+    public static class FlowFactory
+    {
```

Comment:
Is this flow factory a representation of the Factory Pattern?

Reply:
Yes, this FlowFactory represents  the Factory Pattern, as it encapsulates the creation logic for different predefined flows. However, it is designed to dynamically chain and configure test steps specific to this automation framework. You can do the test steps in the main test case but in some scenarios will be redundant repeating the same test steps just for login when you can encapsulate this in the FlowFactory.